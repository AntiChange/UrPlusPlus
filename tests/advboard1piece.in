3 8
*OOOXX*O
OOO*OOOO
*OOOXX*O
0 3
WWWSEEEEEENESSW
2 3
WWWNEEEEEESENNW
1
B 0 3
B 2 3
0
