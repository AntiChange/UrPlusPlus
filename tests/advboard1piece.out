Player 0's turn
Dice Roll: 4295032831, Flex Dice Roll:140721233871120
Board State:
*OOa  *O
OOO*OOOO
*OO1  *O
Player 0 game path:
v--<``>v
>-----^|
``````E<
Player 1 game path:
``````E<
>-----v|
^--<``>^
